solutions/회의실배정.js

```diff
@@ -0,0 +1,31 @@
+const INPUT_PATH = "../inputs/회의실배정.txt"
+
+const fs = require("fs");
+const filePath = process.platform === "linux" ? "/dev/stdin" : INPUT_PATH;
+const input = fs.readFileSync(filePath).toString().trim().split("\n");
+
+function solution(input){
+  const n = parseInt(input[0]);
```

Comment:
```suggestion
  const totalMeeting = parseInt(input[0]);
```

`n`보다는 전체 회의의 갯수를 의미하니까 `totalMeeting`과 같이 조금 더 의미있는 변수명으로 지으면 좋을 것 같습니다!
+
`Number`말고, `parseInt`를 사용하신 이유가 있는지 궁금해요🙂